samples/python/RedisPubSubTrigger/ResolvedChannelPubSubTrigger/function.json

```diff
@@ -4,6 +4,7 @@
       "type": "redisPubSubTrigger",
       "connectionStringSetting": "redisConnectionString",
       "channel": "%pubsubChannel%",
+      "pattern": false,
```

Comment:
Must be "true" ?

Reply:
The `"%pubsubChannel%"` value is not the actual Redis channel that the trigger would subscribe to, but rather the name of the appsetting config where the actual channel is stored. For this sample, the actual channel is resolved from the [`"pubsubChannel"` field in the app settings (`local.settings.json`)](https://github.com/Azure/azure-functions-redis-extension/blob/4c176e6bcd07771a9a7a5216f794a702b769e9e9/samples/python/RedisPubSubTrigger/local.settings.json#L6), and that channel is not a pattern.

Reply:
My bad. Got it. thank you for your work !